includes/class-content-distribution.php

```diff
@@ -47,7 +49,7 @@ public static function init() {
 		}
 
 		add_action( 'init', [ __CLASS__, 'register_data_event_actions' ] );
-		add_action( 'shutdown', [ __CLASS__, 'distribute_queued_posts' ] );
+		add_action( 'shutdown', [ __CLASS__, 'distribute_queued_posts' ], 20 );
```

Comment:
Any reason for the change in priority?

Reply:
No, I think that is a relic from when I was trying to game the timing of things. :) I removed it